frontend-web/webclient/package.json

```
@@ -38,6 +38,7 @@
         "jsrsasign": "11.1.0",
         "localforage": "1.10.0",
         "magic-bytes.js": "1.12.1",
+        "mermaid": "^11.12.1",
```

Please use exact. This is likely `npm` itself that wrote it like this.
```suggestion
        "mermaid": "11.12.1",
```